e2e/kube.go

```diff
@@ -130,7 +127,26 @@ spec:
           privileged: true
           capabilities:
             add: ["SYS_PTRACE", "SYS_RAWIO"]
-`
+`, deploymentName, isHostNetwork, targetNodeLabel)
+}
+
+func createDebugDeployment(ctx context.Context, kube *Kubeclient, manifest string) error {
+	var ds v1.DaemonSet
```

Comment:
wait...

how are we marshaling into a DaemonSet if the manifest specifies kind as `Deployment`?

Reply:
That's a good question, I modified Artur's code here - I was also initally confused by the fact that we have deployment yamls but end up with daemonsets on the cluster. But I sorta ran with it...

Reply:
I didn't write it, just move around. Could be just a typo. Try to fix and see what will happen.

Maybe by coincidence all fields we're using present in both types.

Reply:
Yep fixed it and it seems to be working the same. So naming is consistent now :)